service/frontend/workflowHandler.go

```diff
@@ -2182,7 +2182,7 @@ func (wh *WorkflowHandler) StartWorkflowExecution(
 
 	wh.GetLogger().Debug("Start workflow execution request domainID", tag.WorkflowDomainID(domainID))
 	historyRequest, err := common.CreateHistoryStartWorkflowRequest(
-		domainID, startRequest, time.Now())
+		domainID, startRequest, time.Now(), nil)
```

Comment:
are you intending to add this later? 

I assume we're expecting this field to be optional and going to treat it as such?

Reply:
it will be added later